client/src/components/Admin/Users/List/index.js

```diff
@@ -19,42 +23,73 @@ const UserList = ({ users, toggleCanVote }) => {
-      <thead>
-        <tr>
-          <th className={css.left}>Bruker ({totalUsers})</th>
-          <th className={css.right} title="Har fullført oppmøteregistrering">
-            Registrert ({usersRegistered})
-          </th>
-          <th className={css.right} title="(Antall med stemmerett)">
-            Rettigheter ({usersHasPermsToVote})
-          </th>
-          <th className={css.right}>
-            Stemmeberettigelse ({usersCanVote}/{usersHasPermsToVote})
-          </th>
-        </tr>
-      </thead>
-      <tbody>
-        {Object.keys(users)
-          .sort((a, b) => users[a].name.localeCompare(users[b].name))
-          .map((key) => {
-            const user = users[key];
-            return (<UserContainer
-              name={user.name}
-              canVote={user.canVote}
-              completedRegistration={user.completedRegistration}
-              key={user.id}
-              permissions={user.permissions}
-              registered={user.registered}
-              toggleCanVote={toggleCanVote}
-              id={user.id}
-            />);
+
+  const data = [...Object.keys(users)
+    .sort((a, b) => users[a].name.localeCompare(users[b].name))
+    .map(key => users[key])
+    .map(user => ({
+      name: user.name,
+      canVote: user.canVote,
+      registered: {
+        completedRegistration: user.completedRegistration,
+        registeredDate: user.registered,
+      },
+      id: user.id,
+      permissions: user.permissions,
+      setPermissions: user.setPermissions,
+    }))];
+
+  const columns = [{
+    Header: `Name (${totalUsers})`,
+    accessor: 'name',
+  }, {
+    Header: `Registrert (${usersRegistered})`,
+    accessor: 'registered',
+    className: css.center,
+    Cell: props =>
+      <div
+        className={classNames(css.action,
+          {
+            [css.success]: props.value.completedRegistration,
+            [css.close]: !props.value.completedRegistration,
+            [css.toggle]: props.value.completedRegistration,
           },
-        )}
-      </tbody>
-    </table>
-  );
+            )}
+        title={`${moment(props.value.registeredDate).format('LLL')} (${moment(props.value.registeredDate).fromNow()})`}
+      >{props.value.completedRegistration}</div>,
+  }, {
+    Header: `Rettigheter (${usersHasPermsToVote})`,
+    accessor: 'permissions',
+    Cell: props => getPermissionDisplay(props.value),
+  }, {
+    Header: `Stemmeberettigelse (${usersCanVote}/${usersHasPermsToVote})`,
+    accessor: 'canVote',
+    className: css.center,
+    Cell: props =>
+      <ToggleCanVoteContainer
+        canVote={props.value}
+        id={props.original.id}
+        permissions={props.original.permissions}
+        setPermissions={props.original.setPermissions}
+        toggleCanVote={toggleCanVote}
+      />,
+  }];
+
+  return (
+    <ReactTable
+      data={data}
+      columns={columns}
+      className={css.table}
+      getTrProps={(state, rowInfo, column, instance) => ({
```

Comment:
`instance` is unused here so you can just remove it.

Reply:
Seems like i could remove instance and column. Reason was i couldnt remove state else it wouldnt render even if its unused due to the syntax for getTrTrops